### PR TITLE
New CLI options: --corpus-dir and --check-asserts

### PR DESCRIPTION
**Pull request by arcz**
New CLI options for matching config values. Output from `--help`:
```
Echidna

Usage: echidna-test [--version] FILES [--contract CONTRACT] [--config CONFIG]
                    [--format FORMAT] [--corpus-dir PATH] [--check-asserts]
  EVM property-based testing framework

Available options:
  -h,--help                Show this help text
  --version                Show version
  FILES                    Solidity files to analyze
  --contract CONTRACT      Contract to analyze
  --config CONFIG          Config file
  --format FORMAT          Output format: json, text, none. Disables interactive
                           UI
  --corpus-dir PATH        Directory to store corpus and coverage data
  --check-asserts          Check asserts in the code
```

### CODE REVIEW

**Comment by ggrieco-tob** [comment]
This PR looks good, but I think user should be warned if the command line options and the config ones are overridden, to make sure they know what to expect. Can you add that?

**Comment by arcz** [comment]
CLI switches usually take priority over the corresponding config. Currently this happens for `--format`, `--corpus-dir` and `--check-asserts` and will happen for the rest of switches I'll be implementing. Do you think we should add a note to all of them?

**Comment by arcz** [comment]
@ggrieco-tob added this next to help for `--config` - `Config file (CLI arguments override config options)`

**Review by ggrieco-tob — APPROVED** [review]
Approved, but if possible please fix the CI before merging.

**Pull request merged**
